prov/util/src/util_mem_monitor.c

```diff
@@ -194,6 +194,21 @@ int ofi_monitors_add_cache(struct ofi_mem_monitor **monitors,
 			ret = monitor->start(monitor);
 			if (ret) {
 				cache->monitors[iface] = NULL;
+
+				/* Requested memory monitor is not supported.
+				 * The cache cannot be used for this HMEM type.
+				 */
```

Comment:
We can drop this comment.  The code should make it clear.

Reply:
Sounds good.